libff/algebra/curves/bls12_381/bls12_381.sage

```diff
@@ -0,0 +1,70 @@
+#!/ usr / bin / env sage - python
+
+from sage.all import *import sys
+
+#Prime order of the subgroup we work in
+            def r(x)
+    : return x
+        * *4 -
+    x * *2 +
+    1
+
+#Prime used to generate the base finite field
+        def q(x)
+    : return (x - 1) * *2 *
+    (x * *4 - x * *2 + 1) // 3 + x
+
+#Compute G1 cofactor
+    def g1_h(x)
```

Comment:
Indents need to be fixed

Reply:
Same as above: clang-format messed the formatting.

Reply:
Done